app-editors/bvi/bvi-1.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Display-oriented editor for binary files, based on the vi texteditor"
+HOMEPAGE="http://bvi.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bvi/${P}.src.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~sparc64-solaris ~x86-solaris"
+
+DEPEND="sys-libs/ncurses:0="
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	econf --with-ncurses="${EPREFIX}"/usr
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "make install failed"
```

Comment:
`default`? Also `emake` dies on its own since EAPI 4.

When you do `default`, you can either limit `dodoc` to files not in default list, or define `DOCS` array with complete list of files to install. In the latter case, you'd drop the whole phase and just declare `DOCS` globally.

Reply:
I also noticed that `src_configure` *appears* to be unnecessary (the `--with-ncurses=${EPREFIX}/usr` option just adds `-I/usr/include -I/usr/include -L/usr/lib`, the first and last of which are defaults anyway and the middle of which doesn’t even exist on Gentoo). I assume that, on a prefix build, some sort of magic will be done to get include and lib directories, so I removed `src_configure`, but I’m not sure; if prefix builds need this, please let me know, because I really have no idea how prefix works, and I’ll put it back in.

It turns out the vaguely defined “reasonable default” for `DOCS` seems to include `README`, `CHANGES`, and `CREDITS`, which means I don’t even need to define it. So `src_install` is completely useless and I’ve deleted it. This excludes `bmore.help`, but that goes into `/usr/share/bvi` and I think we don’t need a copy in `/usr/share/doc` because it’s really meant for use within the program.

1407b64e9d00